src/index.js

```diff
@@ -3,17 +3,21 @@ import ReactDOM from "react-dom";
 import App from "./App";
 import { makeServer } from "./server";
 import { BrowserRouter as Router } from "react-router-dom";
-import { FilterProvider } from "./contexts";
+import { AuthProvider, FilterProvider, ProductProvider } from "./contexts";
 
 // Call make Server
 makeServer();
 
 ReactDOM.render(
   <React.StrictMode>
     <Router>
+    <AuthProvider>
+    <ProductProvider>
     <FilterProvider>
       <App />
     </FilterProvider>
+    </ProductProvider>
+    </AuthProvider>
```

Comment:
you can create a router folder and import that one component to her that looks good.